app/models/audit_trail.rb

```diff
@@ -1,18 +1,39 @@
 # Keeps track of the history of objects (mostly patients and pregnancies).
 class AuditTrail
   include Mongoid::History::Tracker
+  include Mongoid::Userstamp
+  mongoid_userstamp user_model: 'User'
+
+  IRRELEVANT_FIELDS = %w(user_ids updated_by)
 
   # convenience methods for clean view display
-  def tracked_changes_fields
-    modified.keys.map(&:humanize)
+  def date_of_change
+    created_at.display_date
```

Comment:
I hear you. Decision from team design was that just date was fine, and it'll still sort in order of change.